model/src/main/proto/topic.proto

```diff
@@ -249,3 +252,35 @@ message CellularDataPreference {
   // Preference on whether to use cellular data
   bool use_cellular_data = 2;
 }
+
+// Corresponds to a single subtopic that can be trained, or reviewed.
+message Subtopic {
+  // The ID of the subtopic.
+  string subtopic_id = 1;
+
+  // The title of the subtopic.
+  string title = 2;
+
+  // The list of skill ids associated with each of the subtopic.
+  repeated string skillIdList = 3;
+}
+
+// Corresponds to a review card that can be displayed for a specific subtopic.
+message ReviewCard {
+  // The title corresponding to the subtopic.
+  string subtopic_title = 1;
+
+  // The subtopic's description that can help provide context on what content is being reviewed.
```

Comment:
Make it simple:
`// Description related to Subtopic`

Reply:
Done.